src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/Local Server/LocalServerMapImageLayer/readme.md

```diff
@@ -17,13 +17,13 @@ The Local Server and local map service will automatically be started and, once r
 1. Create and run a local server with `LocalServer.Instance`.
 2. Start the server asynchronously with `server.StartAsync()`.
 3. Create and run a local service, example of running a `LocalMapService`.
-    1. Instantiate `LocalMapService(Url)` to create a local map service with the given URL path to the map package (`mpkx` file).
+    1. Instantiate `LocalMapService(Url)` to create a local map service with the given URL path to the map package (`mpk` file).
```

Comment:
Is this right?

Reply:
Local Server supports ArcGIS Pro .gpkx and .mpkx files.